releasenotes/notes/nova-machine-type-q35-ab363c5f7f3f9807.yaml

```diff
@@ -0,0 +1,5 @@
+---
+upgrade:
+  - |
+    Configure Nova to use more modern 'q35' libvirt machine type rather than
+    'pc' which is considered legacy.
```

Comment:
Will there be any impact on existing systems from changing this?

Reply:
Most likely no - in yoga all instances should have type metadata already, during upgrade check will fail if not. When restarted new vms should use new type afaik.